packages/@aws-cdk/aws-synthetics/test/code.test.ts

```diff
@@ -0,0 +1,124 @@
+import '@aws-cdk/assert/jest';
+import * as path from 'path';
+import { App, Stack } from '@aws-cdk/core';
+import * as synthetics from '../lib';
+
+let stack: Stack;
+let app: App;
+beforeEach(() => {
+  app = new App();
+  stack = new Stack(app, 'canaries');
+});
+
+describe('Code.fromInline', () => {
+  test('basic use case', () => {
+    // WHEN
+    new synthetics.Canary(stack, 'Canary', {
+      test: synthetics.Test.custom({
+        handler: 'index.handler',
+        code: synthetics.Code.fromInline('exports.handler = async () => {\nconsole.log(\'hello world\');\n};'),
```

Comment:
Doesn't this look better?

```suggestion
        code: synthetics.Code.fromInline(`
          exports.handler = async () => {
            console.log('hello world');
          };
        `),
```